uyuni-site.yml

```diff
@@ -77,6 +77,8 @@ asciidoc:
     sle: SUSE Linux Enterprise
     sleses: SUSE Linux Enterprise Server with Expanded Support
     es: Expanded Support
+    slemicro: 'SLE Micro'
```

Comment:
Come to think, don't we miss this change for the PDF branding file as well?

Reply:
> Come to think, don't we miss this change for the PDF branding file as well?

In entities.adoc, we already have ":micro: SLE Micro", I now also added ":slemicro: SLE Micro".  Maybe, here in .yml we also need "micro: 'SLE Micro", @Loquacity ?

Reply:
Yes, I think `:micro:` is missing here, and `:slemicro:` should go away, as it's not used. Same fur suma-site and the PDFs.

Reply:
another PR created.  I hope I got it right.

Reply:
Oh, I'm so confused now :joy: